Makefile

```diff
@@ -53,3 +56,8 @@ tag: update-kfdef-with-tag
 	git tag $(TAG_NAME)
 push-tag:
 	git push --tags
+
+# requires kube-linter to be installed (https://github.com/stackrox/kube-linter)
+lint:
+	kube-linter lint ./ --config $(KUBE_LINTER_CONFIG)
```

Comment:
Can you add an additional optional argument that would allow us to pass arguments to `kube-linter`?  This would make it easier to target a specific directories instead of the default ALL directories